recipes/kaiju/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 build:
   number: 0  
   no_link:
-    - bin/makeDB.sh -p
+    - bin/makeDB.sh -e
```

Comment:
Please note that `build: no_link:` is just a list of files (or globs), but does not correspond to any command execution.